release notes/v0.46.0.md

```diff
@@ -0,0 +1,95 @@
+
+The new cloud traces will work "out of the box" for `k6 cloud` runs. In case of `k6 run` execution, the `K6_CLOUD_TRACES_ENABLED` environment variable has to be set to `true`.
+
+### `<big_feature_1>` `#pr`
+
+_what, why, and what this means for the user_
+
+### `<big_feature_n>` `#pr`
+
+_what, why, and what this means for the user_
+
+### UX improvements and enhancements
+
+_Format as `<number> <present_verb> <object>. <credit>`_:
+- _`#999` Gives terminal output prettier printing. Thanks to `@person` for the help!_
+
+- TODO: [#3157](https://github.com/grafana/pull/3157) and [#3172](https://github.com/grafana/pull/3172) Add a `MaxTimeSeriesInBatch` configuration option.
+- [#3159](https://github.com/grafana/pull/3159) Adds per-vu TLS configuration to `grpc.Client` connect method's parameters. Thanks @chrismoran-mica 🙇‍♂️.
+- [#3176](https://github.com/grafana/pull/3176) Adds a `js/promises` package, which enables extension developers to easily create promises that will be dispatched to the eventloop using the `New` function.
+- [#3181](https://github.com/grafana/pull/3181) Adds a `RunOnEventLoop` method to the `modulestest.Runtime` type, which allows extensions developers to run code on the event loop from their tests.
+
+## Bug fixes
+
+- [#3163](https://github.com/grafana/pull/3163) Addresses a bug in our gRPC example.
+- [#3178](https://github.com/grafana/pull/3178) Fixes the registration of nested protobuf messages in the gRPC module.
+- [#3170](https://github.com/grafana/pull/3170) Upgrades the version of gRPC k6 depends on to address a security vulnerability ([CVE-2023-32731]).
+
+## Maintenance and internal improvements
+
+### Cloud output v2
+
+// TODO @codebien
+
+* [3104](https://github.com/grafana/k6/pull/3104) Retry and flushers pool
+* [3108](https://github.com/grafana/k6/pull/3108) Flush chunks
+* [3120](https://github.com/grafana/k6/pull/3120) Unlimited size for body payload
+* [3125](https://github.com/grafana/k6/pull/3125) Use a static remote service url
+* [3162](https://github.com/grafana/k6/pull/3162) Warn about labels that have reserved names
+* [3169](https://github.com/grafana/k6/pull/3169) Compact histogram
+- [#3182](https://github.com/grafana/k6/pull/3182) Fix off-by-one error for spans
+- [#3186](https://github.com/grafana/k6/pull/3186) Be less noisy when handling errors
+- [#3187](https://github.com/grafana/k6/pull/3187) Create batches for single bucket flush
+- [#3193](https://github.com/grafana/k6/pull/3193) Downscale the batch size
+- [#3195](https://github.com/grafana/k6/pull/3195) Make sure to use the overwritten config
+- [#3206](https://github.com/grafana/k6/pull/3206) Implement concurrent pushes accross batches
+- [#3226](https://github.com/grafana/k6/pull/3226) don't go over maxSeriesInBatch
+
+### Miscellaneous
+
+- [#3211](https://github.com/grafana/k6/pull/3211) Updates the experimental gRPC module version used by k6.
```

Comment:
As part of another suggestion and rephrasing a bit.

I believe it's important to stress that the [CVE-2023-32731] is related to the demo service, so there is no rush updating to the latest version.

```suggestion
- [#3163](https://github.com/grafana/k6/pull/3163) Fixed gRPC example.
- [#3170](https://github.com/grafana/k6/pull/3170) Upgraded the version of gRPC k6 demo service dependency for addressing a security vulnerability ([CVE-2023-32731]).
- [#3211](https://github.com/grafana/k6/pull/3211) Updates the experimental gRPC module version used by k6.
```